packages/camera/src/components/UploadCenter/index.js

```diff
@@ -151,7 +151,7 @@ export default function UploadCenter({
 
         {/* upload cards */}
         <View style={styles.cardsLayout}>
-          {ids.map((id) => (
+          {ids.reverse().map((id) => (
             <UploadCard
```

Comment:
In my opinion and based on the queue going one by one, I think it is better to reverse the uploads ids order inside the upload center, what do you think @natsukiai ?

Reply:
You are right now that we have a queue. Small details make difference. :-)